README.md

```diff
@@ -132,24 +124,36 @@ To begin registering your app, start at the [Azure portal](https://aka.ms/Mobile
 To **create** an app registration,  
 
 1. Click `New Registration`.
-
 2. Name your app, select the audience you're targeting, and click `Register`.
-
-3. In the `Overview` > `Sign in users in 5 minutes` > `Android`.
-    * Click on `Make this changes for me`.
+3. On the next screen click on `Authentication` and then `Add Platform`.
+4. Select `Android` from the options shown on the right hand side.
+5. On the 'Configure your Android App' page
     * Enter the Package Name from your Android Manifest.
-    * Generate a Signature Hash. Follow the instructions in the portal.
-
-4. Hit the `Make updates` button. Note the ***MSAL Configuration*** as it is used later in `AndroidManifest.xml` and `auth_config.json`.
+    * Enter the Signature Hash. Follow the instructions in the portal to generate Signature hash.
+    * Click `Configure` at the bottom of the page.
+6.  Take note of the ***MSAL Configuration*** as it is used later in `AndroidManifest.xml` and `auth_config.json`.
 
 **Configure** the sample application with your app registration by replacing the sample code in `auth_config.json` and `AndroidManifest.xml`
 
-1. Copy and paste the ***MSAL Configuration*** JSON from the Azure portal into `auth_config.json`.
-
-2. Inside the `AndroidManifest.xml`, replace `android:host` and `android:path` with the same info saved in above step.
-        - `auth_config.json` contains this information as a reference inside the `redirect_uri` field.
-        - The Signature Hash should NOT be URL encoded in the `AndroidManifest.xml`.
-    Refer [Azure Active Directory Android Quickstart](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-android) for more details
+1. Copy and paste the ***MSAL Configuration*** JSON from the Azure portal into `auth_config_multiple_account.json` under the res\raw folder.
+
+2. Inside the `AndroidManifest.xml`, 
+    
+    1. replace `android:host` with the package name
+    
+        ```
+        android:host="<packaage_name>"
+        ```
+    
+    2.  replace `android:path` with signature hash.
+    
+        ```
+        android:path="<SignatureHash>"
+        ```
+    
+            - `auth_config.json` contains this information as a reference inside the `redirect_uri` field.

```

Comment:
Formatting here is causing this bullet point to render strangely

Reply:
![Capture](https://user-images.githubusercontent.com/990063/65927530-e1958d00-e3ae-11e9-8938-b62fb8a9fc1e.PNG)

Reply:
Fixed. Thanks for the image :)